### ISSUE TITLE
[ui5-switch]: DIGMANEXE-406098 ACC2511_ACC 261.1(Level AA) _ DMC 2511_ Text _ Does not meet contrast ratio in SWITCH

### ISSUE THREAD

**Issue by i057810** [UI5/webcomponents#12609]
### Bug Description

Observed Behavior:
‘Yes’ text information is not displayed with a contrast ratio of at least 4:5:1 ratio in default theme.

### Expected Behaviour

Expected Behavior: 
In default theme (Quartz Light), texts and images of texts smaller than 18 pt (or 14 pt bold) should have minimum contrast ratio of 4.5:1.

### Steps to Reproduce

1.
2.
3.
...

### Priority

Low

### UI5 Web Components Version

https://sapui5.hana.ondemand.com/1.136.9/

### Browser

Chrome

### Additional Context

Observed Behavior:
‘Yes’ text information is not displayed with a contrast ratio of at least 4:5:1 ratio in default theme.
Expected Behavior: 
In default theme (Quartz Light), texts and images of texts smaller than 18 pt (or 14 pt bold) should have minimum contrast ratio of 4.5:1.

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by i057810** [comment]
<img width="482" height="283" alt="Image" src="https://github.com/user-attachments/assets/b68bc8a6-125c-4f69-bc65-6b82a2c09a52" />

**Comment by nnaydenow** [comment]
Hi @i057810,

Is the issue related to OpenUI5 / SAPUI5 or related to UI5 Web Components?

If it's related to OpenUI5 / SAPUI5  please submit the issue in: https://github.com/UI5/openui5/

**Comment by i057810** [comment]
Thanks
